src/util/uint.rs

```diff
@@ -215,15 +215,15 @@ macro_rules! construct_uint {
 
             #[inline]
             fn sub(self, other: $name) -> $name {
-                self + !other + $crate::util::BitArray::one()
+                self + !other + $crate::util::uint::BitArray::one()
```

Comment:
Maybe flatten `uint` first? Not a big deal though.

Reply:
`uint` will go away with the [target/work PR](https://github.com/rust-bitcoin/rust-bitcoin/pull/1197#issuecomment-1228569693). I guessed this one would go in first and I'd rebase the other one.